src/topp/IDMerger.cpp

```diff
@@ -281,7 +281,7 @@ class TOPPIDMerger :
 
       if (!add_to.empty())
       {
-        remove(file_names.begin(), file_names.end(), add_to);
+        file_names.erase(std::remove(file_names.begin(), file_names.end(), add_to), file_names.end());
```

Comment:
@hendrikweisser @timosachsenberg agreed?

Reply:
Let's see. Now, add_to it is actually removed.
Before your change, the add_to filename was only moved to the end of file_names.
In the next line add__to is added at the front of file_names.
Need to check the downstream code because both ways could (in principle) make sense.

Reply:
yep your change seems correct